ESCPOS_NET/Printers/BasePrinter.cs

```diff
@@ -1,51 +1,71 @@
 ﻿using System;
-using System.Collections;
-using System.Collections.Concurrent;
+using System.Collections.Concurrent;
+using System.Diagnostics;
 using System.IO;
 using System.Linq;
-using System.Text;
 using System.Threading;
 using System.Threading.Tasks;
-using System.Timers;
+using System.Timers;
 
 namespace ESCPOS_NET
 {
     public abstract partial class BasePrinter : IDisposable
     {
-        public PrinterStatus Status = null;
+    	private bool disposed = false;
```

Comment:
this indentation looks off in this section

Reply:
fixed